src/plugins/release-monitor/index.js

```diff
@@ -224,14 +179,7 @@ async function issueLabeledHandler(context) {
 
     // check if the label is post-release and the same issue has release label
     if (isPostReleaseLabel(context.payload.label) && hasReleaseLabel(context.payload.issue.labels)) {
-
-        // put pending status on every PR which doesn't have fix or docs status.
-        await createStatusOnPRs({
-            context,
-            prs: await getNonSemverPatchPRs(context),
-            isSuccess: false,
-            releaseIssueUrl: context.payload.issue.html_url
-        });
+        await createStatusOnAllPRs({ context, pendingReleaseIssueUrl: context.payload.issue.html_url });
```

Comment:
multi-line objects are more readable. I would recommend changing it to old style.